control/control.go

```diff
@@ -1035,20 +1035,20 @@ func (r *requestedPlugin) Config() *cdata.ConfigDataNode {
 // ------------------- helper struct and function for grouping metrics types ------
 
 // just a tuple of loadedPlugin and metricType slice
-type pluginMetricTypes struct {
+type metricTypes struct {
```

Comment:
Overloading types here for the different packages. Is it a good practice in Golang?